services/web/client/source/class/qxapp/auth/core/BaseAuthPage.js

```diff
@@ -53,6 +53,19 @@ qx.Class.define("qxapp.auth.core.BaseAuthPage", {
      */
     _buildPage: null,
 
+    /**
+     * This method needs to be implemented in subclass
+     * and should reset all field values
+    */
+    resetValues: function() {
+      this.getChildren().forEach(item => {
+        // FIXME: should check issubclass of AbstractField
+        if (qx.Class.implementsInterface(item, qx.ui.form.IForm) && qx.Class.implementsInterface(item, qx.ui.form.IField)) {
+          item.resetValue();
+        }
+      });
+    },
+
     /**
```

Comment:
you can also just reset the form ...

Reply:
I understand your point.... I thought about it but went (for the moment) for the easiest solution.

Yes, forms have a ``resetValue`` because they implement the [qx.ui.form.IField]() interface . Unfortunately, there are classes that do not implement [qx.ui.form.IForm]() interface but still implement the former, e.g.  ``qx.ui...Labels`` ... and I do not want to reset the latter.  

In addition, my classes do not have forms as immediate children but can be one or two generations deeper. ... which makes it slightly more complex.

Reply:
qooxdoo has a form infrastructure which works quite well ... and if I need special looks, I implement a custom form renderer (like I did for the settings view) ... but yes it is certainly possible to take another route ...